app/Annotations/OpenApi/controllersAnnotations/studentAnnotations/AnnotationsStudentDetail.php

```diff
@@ -5,66 +5,63 @@
-     *                             ),
-     *                             @OA\Property(
-     *                                 property="github",
-     *                                 type="object",
-     *                                 @OA\Property(property="url", type="string", example="https://github.com/"),
-     *                             )
+     *                             @OA\Property(property="url", type="string", example="https://github.com/bettie52")
      *                         ),
      *                         @OA\Property(
-     *                             property="about",
+     *                             property="linkedin",
      *                             type="object",
-     *                             @OA\Property(property="description", type="string", example="Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nam a euismod elit. Nunc elit ante, vulputate sed maximus eu, posuere eu nibh. In eget lacus in mi pharetra volutpat a a justo. Fusce aliquet nibh nec accumsan facilisis. Suspendisse in tempor nibh, eu fermentum velit. Suspendisse cursus ultricies ipsum, eget tincidunt arcu pretium laoreet. Pellentesque eget egestas erat. Donec dapibus pharetra ultrices. Vivamus mollis sapien sed laoreet interdum."
-     *                              )
-     *                          ),
-     *                             @OA\Property(property="tags",type="array",@OA\Items(ref="#/components/schemas/Tag"))
+     *                             @OA\Property(property="url", type="string", example="https://linkedin.com/abernathy.dayne")
      *                         )
-     *
-     *                  )
-     *              )
-     *          )
-     *      ),
-     *
-     *
-     *      @OA\Schema(
-     *           schema="Tag",
-     *           type="object",
-     *           @OA\Property(property="id", type="integer", example=1),
-     *           @OA\Property(property="name", type="string", example="JavaScript"),
-     *           example={"id": 1, "name": "JavaScript"}
-     *      )
+     *                     ),
+     *                     @OA\Property(property="about", type="string", example="Iusto aut debitis soluta facere tempore quisquam. Vel assumenda aliquid quod et eum quos ex. Ipsa ea tempora minima occaecati. Culpa occaecati quod laboriosam reiciendis quia consequuntur."),
+     *                     @OA\Property(
+     *                         property="tags",
+     *                         type="array",
+     *                         @OA\Items(
+     *                             @OA\Property(property="id", type="integer", example=4),
+     *                             @OA\Property(property="name", type="string", example="HTML&CSS")
+     *                         )
+     *                     )
+     *                 )
+     *             )
+     *         )
+     *     ),
+     *     @OA\Response(
+     *         response=404,
+     *         description="Usuario no encontrado."
```

Comment:
Here, you could use the content of the message that throws your exception, which is more descriptive.

Reply:
Done